SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -521,8 +509,8 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
 
     public func exchange(tokenV1: String) async throws -> TokenContainer {
         let tokenContainer = try await oAuthClient.exchange(accessTokenV1: tokenV1)
-        cachedIsUserAuthenticated = true
-        cachedUserEntitlements = tokenContainer.decodedAccessToken.subscriptionEntitlements
+            cachedIsUserAuthenticated = true
+            cachedUserEntitlements = tokenContainer.decodedAccessToken.subscriptionEntitlements
```

Comment:
Indentation seems off here and in a few lines below too.

Reply:
fixed